### PR TITLE
Init translation of ariaLive

### PR DESCRIPTION
**Pull request by hanyujie2002**
### Description
Init translation of ariaLive

### Motivation
Make other Chinese users easier to understand it 

### Additional details
none

### Related issues and pull requests
none

### CODE REVIEW

**Comment by hanyujie2002** [comment]
I rebase my local branch on `upstream/main` so there are only relative commits shown

File: files/zh-cn/web/api/element/arialive/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: 元素：ariaLive 属性
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/arialive/index.md:2]: 这里的 Element 是[对应接口](https://developer.mozilla.org/en-US/docs/Web/API/Element)的名称，倾向于不译

```suggestion
title: Element：ariaLive 属性
```

File: files/zh-cn/web/api/element/arialive/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- `"assertive"`
+  - : 表示对该区域的更新具有最高优先级，并应立即呈现给用户。
+- `"off"`
+  - : 表示除非用户当前关注该区域，否则不应向用户呈现对该区域的更新。
+- `"polite"`
+  - : 表示应在下一个适当的机会（例如，在说完当前句子或用户暂停输入时）呈现对该区域的更新。
+
+## 示例
+
+在此示例中，ID 为 `planetInfo` 的元素上的 [`aria-live`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-live) 属性设置为 "polite"。然后我们将值更新为"assertive"。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/arialive/index.md:23]: [请使用全角标点符号](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#标点符号)

```suggestion
在此示例中，ID 为 `planetInfo` 的元素上的 [`aria-live`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-live) 属性设置为“polite”。然后我们将值更新为“assertive”。
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/api/element/arialive/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: 元素：ariaLive 属性
+slug: Web/API/Element/ariaLive
+---
+
+{{DefaultAPISidebar("DOM")}}
+
+**`ariaLive`** 属性是 {{domxref("Element")}} 接口的一部分，它反映了 [`aria-live`](/zh-CN/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) 属性的值，该属性表示一个元素将被更新，并描述了用户代理、辅助技术和用户可以从实时区域中期望的更新类型。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/arialive/index.md:8]: https://github.com/mdn/content/pull/28656/

```suggestion
**`ariaLive`** 属性是 {{domxref("Element")}} 接口的一部分，它反映了 [`aria-live`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-live) 属性的值，该属性表示一个元素将被更新，并描述了用户代理、辅助技术和用户可以从[实时区域](/zh-CN/docs/Web/Accessibility/ARIA/ARIA_Live_Regions)中期望的更新类型。
```

File: files/zh-cn/web/api/element/arialive/index.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  <h2 id="planetTitle">未选择星球</h2>
+  <p id="planetDescription">选择一个星球以查看其描述</p>
+</div>
+```
+
+```js
+let el = document.getElementById("planetInfo");
+console.log(el.ariaLive); // "polite"
+el.ariaLive = "assertive";
+console.log(el.ariaLive); // assertive
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/arialive/index.md:36]:
```suggestion
console.log(el.ariaLive); // "assertive"
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**